### PR TITLE
Add resolution sizes, stylelint & base font-size

### PR DESCRIPTION
**Pull request by ao-armen**

### CODE REVIEW

**Comment by ao-tigran** [comment]
@hpilosyan Should we merge this? There seem to be no comments.

**Comment by ao-tigran** [comment]
@hpilosyan please comment on this one.

File: README.md [hunk 1]
```diff
@@ -1,3 +1,12 @@
+For cases when writing `px` is necessary write your line between `/* stylelint-disable */` and `/* stylelint-enable */`
```
Comment (hpilosyan) [inline README.md]: Please cleanup the readme and use proper markdown

File: package.json [hunk 2]
```diff
@@ -16,7 +16,8 @@
     "start": "react-scripts start",
     "build": "react-scripts build",
     "test": "react-scripts test",
-    "eject": "react-scripts eject"
+    "eject": "react-scripts eject",
+    "lint": "stylelint src/**/*.scss !**/src/.scss/**"
```
Comment (hpilosyan) [inline package.json]: Should we run this manually integrate husky and lint-staged

File: src/components/LoginScreen/index.js [hunk 3]
```diff
@@ -14,6 +15,8 @@ const LoginScreen = props => {
     <div id={styles.login_screen}>
       <div className={styles.login_body}>
         <form onSubmit={handleFormSubmit}>
+          <LanguageContainer />
```
Comment (hpilosyan) [inline src/components/LoginScreen/index.js]: Why do you need this?

File: src/components/LoginScreen/index.module.scss [hunk 4]
```diff
@@ -4,32 +4,34 @@
...
 
       input {
-        height: 40px;
-        width: 280px;
-        padding-left: 7px;
+        height: 2.5em;
+        width: 15em;
+        padding-left: 0.5em;
+        /* stylelint-disable */
         border: 1px solid #cfd8e6;
```
Comment (hpilosyan) [inline src/components/LoginScreen/index.module.scss]: No way to avoid px?

Comment (hpilosyan) [reply src/components/LoginScreen/index.module.scss]: Or could we extract all cases of using px into a separate location?

**Comment by ao-tigran** [comment]
@ao-armen rebase

**Comment by hpilosyan** [comment]
Rebase

**Pull request merged**